src/astro/databases/sqlite.py

```diff
@@ -0,0 +1,101 @@
+import pathlib
+from typing import Optional, Union
+
+import pandas as pd
+from airflow.providers.sqlite.hooks.sqlite import SqliteHook
+from sqlalchemy import create_engine, text
+from sqlalchemy.engine.base import Engine
+from sqlalchemy.engine.result import ResultProxy
+
+from astro.constants import LoadExistStrategy, MergeConflictStrategy
+from astro.databases.base import Database as BaseDatabase
+from astro.sql.tables import Table
+
+
+class Database(BaseDatabase):
```

Comment:
This should be called one of the above so we don't have name Collisions instead please

```suggestion
class Sqlite(BaseDatabase):
```

or 

```suggestion
class SqliteDatabase(BaseDatabase):
```
or

```suggestion
class SqliteDB(BaseDatabase):
```

Reply:
@kaxil I did think about potential name collision issues. However, given the current approach to load the database and the fact we don't aim to expose these classes to the end-users (as opposed to existing Airflow hooks), in which circumstances do you see collision issues?

Reply:
Yeah, I still feel it is confusing to have just `Database` within each module. In my opinion, it should be clear from the class name itself on what it is, which we won't in this case without modules path. That way we won't even need to have `from astro.databases.sqlite import Database as SqliteDatabase` in tests and other places and it is clear

Reply:
@kaxil thanks for your feedback, I addressed it :)